config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml

```diff
@@ -20,7 +20,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.6
```

Comment:
could probably leave the 1.25 jobs alone, since 1.25 is using go1.19

Reply:
right, and I expect 1.24 will never use go 1.19, so these reversions to the 1.24 jobs should be durable

@marosset PTAL